tests/testthat/test-baselinenowcast.R

```diff
@@ -183,6 +202,96 @@ test_that("assert_baselinenowcast_df errors when appropriate", {
   )
 })
 
+test_that("baselinenowcast.data.frame returns the expected structure with and without strata sharing", { # nolint
+  nowcasts_df <- baselinenowcast(
+    data = covid_data,
+    max_delay = 40,
+    draws = 100,
+    strata_cols = c("age_group", "location")
+  )
+
+  expect_s3_class(nowcasts_df, "data.frame")
```

Comment:
Again basicallyt the same code here with an arg shift that could be in a function